wagtail/images/static_src/wagtailimages/js/focal-point-chooser.js

```diff
@@ -30,6 +30,20 @@ function setupJcrop(image, original, focalPointOriginal, fields) {
       // Set alt="" on the image so its src is not read out loud to screen reader users.
       var $holderImage = $('img', jcropapi.ui.holder);
       $holderImage.attr('alt', '');
+
+      // Create a label tag for the input for accessibility.
+      const label = document.createElement('label');
+
+      label.setAttribute('for', 'jcrop');
+      label.classList.add('className', 'visuallyhidden');
+      label.innerHTML = 'Cropping Tool';
```

Comment:
Are we able to use the JavaScript translation system here, it would be good to have this translatable.

Also it should probably be lower case 'tool' so 'Cropping tool'.

Reply:
You can use `window.getText` for this - search through the code for `getText` to see usage.

See https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#internationalization-in-javascript-code for more information in the Django docs for what this is / how it works.